oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -128,6 +186,11 @@ int mca_spml_ucx_del_procs(ompi_proc_t** procs, size_t nprocs)
 
         /* mark peer as disconnected */
         mca_spml_ucx_ctx_default.ucp_peers[i].ucp_conn = NULL;
+        /* release the cached_ep_mkey buffer */
+        for(j=0; j < mca_spml_ucx_ctx_default.ucp_peers[i].mkeys_cnt; j++) {
+            mca_spml_ucx_ep_mkey_release(&(mca_spml_ucx_ctx_default.ucp_peers[i]), j);
+        }
```

Comment:
```suggestion
```
I was thinking of using  mca_spml_ucx_ep_mkey_release() in a place where you destroy ucp rkey

Reply:
I see you use it there as well. What hapens to the ucp rkeys that remain here? Should we release them as well?

Reply:
we destroy ucp rkey in 3 places, deregister, rmkey_free and _ctx_cleanup. _ctx_cleanup release the buffer registered during create common (shmem_ctx_create). Also, as the seg_id and peer information is missing in rmkey_free and deregister, we put release function inside del procs to cleanup all the local mkey buffer which are related to default context.